src/client/browserContext.ts

```diff
@@ -92,6 +92,10 @@ export class BrowserContext extends ChannelOwner<channels.BrowserContextChannel,
     this._channel.setDefaultTimeoutNoReply({ timeout });
   }
 
+  browser(): Browser {
+    return this._browser!;
```

Comment:
not sure about the `!`. There should be always a browser instance.

Reply:
We should probably change the type of this field to `Browser` and throw in the constructor if `parent instanceof Browser` check fails.